install/upgrade/update_to_dev.php

```diff
@@ -163,6 +163,12 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addField($table, 'apple_max_version', 'bool', ['after' => 'apple_min_version']);
    $migration->addKey($table, 'group', 'group');
    $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
+   if ($DB->fieldExists($table, 'is_android_policy')) {
+      $migration->dropField($table, 'is_android_policy');
+   }
+   if ($DB->fieldExists($table, 'is_apple_policy')) {
+      $migration->dropField($table, 'is_apple_policy');
+   }
```

Comment:
Migration::dropField() already checks if the field exists before actually dropping it. This change is useless